### PR TITLE
chore: remove `Slider` outline

### PR DESCRIPTION
**Pull request by Wxh16144**
the `::before` pseudo-element here seems to make no sense and gives an incorrect border when setting the `:focus-visible` style.

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

ref: https://github.com/ant-design/ant-design/issues/38463#issuecomment-1308497086
fix: https://github.com/ant-design/ant-design/issues/38490

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    remove `Slider` outline       |
| 🇨🇳 Chinese |    移除 `Slider` outline      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by MadCcc** [comment]
这样不行，外面的蓝圈是 box-shadow，没有 before 选区太小了

**Comment by Wxh16144** [comment]
> 这样不行，外面的蓝圈是 box-shadow，没有 before 选区太小了

image

这里的蓝色位移应该设计不是如此吧。不过还是感觉这个 before 没有存在意义呢

**Comment by MadCcc** [comment]
before 就是扩大选区用的，如果位置不对应该调整一下

**Comment by MadCcc** [comment]
另外还有一个问题是 hover 轨道的时候蓝圈不应该变大的，可以一起修一下

**Comment by Wxh16144** [comment]
ok我这边已经处理完成

**Comment by MadCcc** [comment]
lint 挂了

File: components/slider/style/index.tsx [hunk 1]
```diff
@@ -108,16 +131,22 @@ const genBaseStyle: GenerateStyle<SliderToken> = token => {
...
+              (token.handleSizeHover - token.handleSize) / 2 +
+              token.handleLineWidthHover
+            ),
             width: token.handleSizeHover + token.handleLineWidthHover * 2,
             height: token.handleSizeHover + token.handleLineWidthHover * 2,
           },
 
           '&::after': {
-            boxShadow: `0 0 0 ${token.handleLineWidthHover}px ${token.colorPrimary}`,
+            boxShadow: `0 0 0 ${token.handleLineWidthHover}px ${token.colorPrimaryHover}`,
```
Comment (MadCcc) [inline components/slider/style/index.tsx:149]:
```suggestion
            boxShadow: `0 0 0 ${token.handleLineWidthHover}px ${token.colorPrimary}`,
```

**Review by MadCcc — APPROVED** [review]

**Pull request merged**